testsuite/features/secondary/min_recurring_action.feature

```diff
@@ -0,0 +1,279 @@
+# Copyright (c) 2020 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+@scc_credentials
+Feature: Recurring Actions
+
+  Scenario: Create a minion Recurring Action
+    Given I am authorized as "admin" with password "admin"
+    When I am on the "States" page of this "sle_minion"
```

Comment:
When -> And
Same in the other scenarios.

Reply:
I'd say "When" is fine here.

Reply:
I'm just curious, @Bischoff why? :thinking: 
We agreed that `When` is for actions, like I click, I follow, I pick, I wait, etc.
And almost all our `I am on the` steps, are using `Given`. Why we change of mind here?